website/docs/contributing/writing-guide.md

```diff
@@ -77,6 +77,39 @@ Rather than use relative paths, we strongly suggest the following approach, sinc
 
 `![FerretDB logo](/img/logo-dark.png)`.
 
+## Tags
```

Comment:
Could we also briefly describe keywords, please?

Reply:
Thanks. I'll add it.